webpack.config.js

```diff
@@ -279,6 +279,12 @@ module.exports = (env, argv) => {
         publicPath: '.',
         template: 'src/authoring/index.html'
       }),
+      ...configHtmlPlugins({
+        chunks: ['iframe'],
+        filename: 'authoring-iframe/index.html',
+        publicPath: '.',
```

Comment:
I'd recommend removing the publicPath here and in the authoring/index.html one above. The publicPath is set by configHtmlPlugins